.ci/.e2e-tests.yaml

```
@@ -26,6 +26,9 @@ SUITES:
       - name: "Linux Integration"
         pullRequestFilter: " && ~debian"
         tags: "linux_integration"
+      - name: "System Integration"
```

This will override the parent platforms element, as the item has precedence:
```suggestion
      - name: "System Integration"
        platforms:
          - "ubuntu-18.04"
```

---

done